gef.py

```diff
@@ -7966,19 +7966,22 @@ def print_guessed_arguments(self, function_name: str) -> None:
 
         def __get_current_block_start_address() -> int | None:
             pc = gef.arch.pc
+            max_distance = 10 * 16
```

Comment:
maybe say something like max_distance_bytes so we know the unit (vs max instructions)